### DISCUSSION TITLE
Thank you and see you next year

### DISCUSSION THREAD

**Discussion by MisRob** [learningequality/kolibri#11646] [Community]
Hello friends,

Before the end of 2023, I'd like to take a moment to reflect on the past few months when our core team has had many beautiful opportunities to cooperate closely with the open-source community.

We've received so many wonderful contributions from you such as bug fixes, accessibility improvements, developer documentation enhancements, new development tooling, components for our design system, maintenance tasks, and much more. Not only our developers, but the whole team has regular opportunities to see and celebrate contributions and everyone is excited! We'd like to express our gratitude and appreciation to all of you.

Personally, these past months have been so fulfilling for me. I've learned a lot from being in touch with you. Seeing that there are so many talented, hard-working people volunteering their time, skills, and energy to support accessible, free, and high-quality educational content and technology is such an inspiration. My hope is that we can engage with you in ways that are meaningful for you too and continue growing together next year. I'd like to also thank the Learning Equality team for their openness and support, whether when directly engaging with contributors, or when exploring internal processes that help us to do so.

And what's next? We're very much looking forward to future cooperation. In January, we will start evaluating the survey that some of the contributors submitted. We will locate areas where we can improve our engagement as well as ways of acknowledging volunteers. We are also planning to keep improving our tooling so that we don't miss a message! 

I wish you all joyful and fulfilling transition to 2024. Please get plenty of rest and let's do even more great work in 2024!

Much gratitude to all of you and I'm looking forward to next year by your side

On behalf of Learning Equality
Michaela (software engineer)

**Comment by MisRob** [comment]
Thank you @a6ar55 @adviti-mishra @akash5100 @Akila-I @andreamisuraca @BabyElias @cerdo03 @dbnicholson @Devanshu-Augusty @dylanmccall @EshaanAgg @FireSuperior482 @GarvitSinghal47 @Ghat0tkach @haldaranup @im-NL @indirectlylit @Jaspreet-singh-1032 @katkuskris @KshitijThareja @laynestephens @mhasbini @muditchoudhary @moweiss @nikkuAg @pwithnall @Shashank245 @ShivangRawat30 @Soren568 @thesujai @Tweniee @yash1378

**Comment by MisRob** [comment]
Also please note Learning Equality will be closed for the holidays from December 22 to January 2, and even as we're transitioning to and from this period of rest we may be slow to respond.

**Comment by marcellamaki** [comment]
So beautifully said, Misha. Thank you so much to all of our contributors! We appreciate you sharing your time and talents with Learning Equality, helping to make our products better for all of our users. Looking forward to continuing to work with you in the new year!

**Comment by rtibbles** [comment]
Very grateful to all our open source contributors - you make a huge difference to the students, teachers, and administrators who use Kolibri - fixing bugs that might otherwise be neglected, improving the overall quality of the code to ensure that new functionality can be created more rapidly, and even creating new functionality too.

It's been a huge joy to see all the excitement and engagement from our open source community, and I look forward to more in 2024!

Thank you all so much!

**Comment by bjester** [comment]
A very big thank you to all community contributors for your support this year!

**Comment by vkWeb** [comment]
@MisRob Amazingly written! :heart: 

I personally feel so delighted when I see so much impact being created by our open source volunteers. Being a volunteer, you guys have done a lot of hard work to figure things out around our codebases by your own selves, proud of you all. Extremely excited to see what we all do together in 2024. :rocket: We @ LE will keep on improving our processes to better support our community. Thank you all :handshake:
